lib/tokens.js

```diff
@@ -40,25 +40,31 @@ const idEncoder = new IdEncoder({
 
 bedrock.events.on('bedrock-mongodb.ready', async () => {
   await promisify(database.openCollections)([
-    'tokenization-tokenBatch', 'tokenization-pairwiseToken'
+    'tokenization-tokenBatch', 'tokenization-openTokenBatch',
+    'tokenization-pairwiseToken'
   ]);
 
   await promisify(database.createIndexes)([{
-    // token batches are sharded by `tokenBatch.id` -- can't shard by
-    // `internalId` because it will not be known at query time because it is
-    // not shared externally and can't make this index unique because the
-    // collection is sharded on a field that isn't part of it; however,
-    // uniqueness is handled by the index below and by the randomness property
-    // of batch IDs
+    /* Note: The `tokenBatch` collection is sharded by `tokenBatch.id`. This
```

Comment:
this might read 'should be sharded' since the shards are setup outside of code here.